packages/web-react/src/components/Container/demo/ContainerDefault.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ContainerDefault = () => {
+  return <div className="docs-Box">Container</div>;
```

Comment:
Implicit return, please. And the `DocsBox` component as I suggested in GridSpan PR. Maybe here it will be much quicker ;-)

Reply:
Changed return implicit and added `<DocsBox />` component ✨

Reply:
Great! Nice and shiny 💅 👍 Let's merge this :-)